src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2407,6 +2396,18 @@ public Response getContentAccessBody(@Verify(Consumer.class) String consumerUuid
             throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate"));
         }
 
+        OffsetDateTime sinceOffsetDateTime = since != null ?
+            Util.parseOffsetDateTime(SINCE_DATE_FORMATER, since) :
+            null;
+
+        Date sinceDate = sinceOffsetDateTime != null ? Util.toDate(sinceOffsetDateTime) : new Date(0);
+        Date scaCertificateLastUpdated = cac.getUpdated() != null ? cac.getUpdated() : new Date();
```

Comment:
Note that this will change when we retrieve the individual components as part of CHAINSAW-359 instead of using the `scaCertificateGenerator.generate` method.